indico/modules/rb_new/client/js/modules/calendar/Calendar.jsx

```diff
@@ -183,15 +218,17 @@ export default connect(
     state => ({
         isFetching: calendarSelectors.isFetching(state),
         calendarData: calendarSelectors.getCalendarData(state),
+        filters: calendarSelectors.getFilters(state),
         rooms: roomsSelectors.getAllRooms(state),
```

Comment:
i think this one might be unused - can you check and remove it if that's the case?

Reply:
Filters are being used in the componentDidUpdate function.